src/render.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::util;
+use vello::peniko::{BlendMode, Fill};
+use vello::Scene;
+
+pub(crate) fn render_group<F: FnMut(&mut Scene, &usvg::Node)>(
```

Comment:
Yes - but the default error handler which paints red boxes over unsupported nodes never panics, it's Infallible. 

If a user wants to panic, they should provide their own error handler.

The need for a result here was never used and seems unnecessary because most users would want to panic or draw nothing (instead of the red box).

Reply:
Yes. The ability which the error represented was "this is an expected condition, but we handle it by not drawing the svg at all". But as I say, using `catch_panic` for that use case is *fine*